pb/api/v1/data.proto

```diff
@@ -162,6 +167,85 @@ message ListRowsResponse {
   }
 }
 
+message GetDashboardTabRequest {
+  string scope = 1;
+  string dashboard = 2;
+  string tab = 3;
+}
+
+message GetDashboardTabResponse {
+
+  // GCSConfig specifies results stored in GCS, typically created by prow.
+  message GCSConfig {
+    // Path to the test result stored in gcs (some-bucket/some/optional/path).
+    string gcs_prefix = 1;
+  
+    // The GCP project where GCS sends notifications for the above prefix.
+    string pubsub_project = 2;
+    // The pubsub subscription ID in the above topic
+    string pubsub_subscription = 3;
+  }
+
+  // A single tab on a dashboard.
+  message DashboardTab {
```

Comment:
This can be unnested, the response is for a DashboardTab so we don't need to note that it's a dashboard tab ^^